src/ui/section/EditDriverTask.spec.tsx

```diff
@@ -183,6 +183,7 @@ describe('EditDriverTask', () => {
       week: 1,
       location: 'New York',
       userID: 0,
+      description: 'Default description',
```

Comment:
Can you also add a `fireEvent.change` like the ones above this statement and have it enter a description besides "Default description" and change this expectation to match?